CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 Magento Functional Testing Framework Changelog
 ================================================
+3.0.0
+-----
+### Upgrade Instructions
+* Run `bin/mftf reset --hard` to remove old generated configurations.
+* Run `bin/mftf upgrade:tests`. [See command page for details](./docs/commands/mftf.md#upgradetests).
+* After running the above command, you will need to manually update some tests:
+    * Remove all occurrences of `<executeInSelenium>` and `<performOn>`
```

Comment:
There is another possible manual step:
If there are any references of <module file=""> in suite XML files, follow console instruction to change them to use <test name=""> instead.

Reply:
Adding 👍